src/pages/Delivery/index.jsx

```diff
@@ -31,17 +31,16 @@ const Delivery = () => {
     "ready": { headerTitle: 'Listo para salir', headerSubtitle: 'El paquete se encuentra listo para salir...' },
     "collected": { headerTitle: 'Recolectado', headerSubtitle: 'El paquete fue recogido por el Dasher...' },
     "in_travel": { headerTitle: 'En camino', headerSubtitle: 'Vamos con tu envio...' },
-    "destination_reached": { headerTitle: 'Destino alcanzado', headerSubtitle: 'Hemos llegado' }
+    'destination_reached': { headerTitle: 'Destino alcanzado', headerSubtitle: 'Hemos llegado' },
+    'rated': { headerTitle: 'Destino alcanzado', headerSubtitle: 'Hemos llegado' },
+    'destination_confirmed': { headerTitle: 'Destino alcanzado', headerSubtitle: 'Hemos llegado' },
```

Comment:
Good job adding it. Please try to use only one type of quotes

```
    "ready": { headerTitle: 'Listo para salir', headerSubtitle: 'El paquete se encuentra listo para salir...' },
    "collected": { headerTitle: 'Recolectado', headerSubtitle: 'El paquete fue recogido por el Dasher...' },
    "in_travel": { headerTitle: 'En camino', headerSubtitle: 'Vamos con tu envio...' },
```
have double quotes and others on this block of code have single quotes